packages/analytics/lib/index.d.ts

```diff
@@ -324,6 +324,17 @@ export namespace FirebaseAnalyticsTypes {
     transaction_id?: string;
   }
 
+  export interface ScreenViewParameters {
+    /**
+     * Screen name the user is currently viewing.
```

Comment:
@mikehardy Good catch. Question: Do you think we should change it to "Set to `null` to clear the current screen name" to keep it inline with iOS & Android? Web [docs](https://firebase.google.com/docs/reference/js/firebase.analytics.Analytics#eventname:-screen_view) don't reveal if setting to `null` clears the current screen unfortunately.

Reply:
Good point - better to be explicit and do null vs undefined (a, javascript...) to match exactly. I rooted around through JS SDK and it was inconclusive as well - I think that might just be because the function call with a null to clear in the web context simply makes no sense, it doesn't map to the paradigm. On native you might want to clear it because they auto-log screen events if you switch activities or view etc but (on react-native) you never switch thus the manual calls are needed, whereas on web you always have to call it or it never gets called? Not sure what the summary is there other than I like the idea of specific null to clear, matching both natives, and if web doesn't do exactly the same it's probably fine because logging a null screen is silly on web to begin with?

Reply:
@mikehardy Makes sense to me 😄

Reply:
I think it's best to leave as it is then? 🤔

Reply:
That's a tough one. I think that's actually a documentation fail on google's part as I bet the underlying behavior is the same as previously documented, but better to conform to their documentation - leave as is I think